star-ui/src/components/AlertsPanel.tsx

```
@@ -1,22 +1,39 @@
 };
 
-const LEVEL_NAMES: Record<number, string> = {
-  [AlertLevel.UNKNOWN]: 'UNK',
-  [AlertLevel.INFO]: 'INFO',
-  [AlertLevel.WARN]: 'WARN',
-  [AlertLevel.ERROR]: 'ERR',
-  [AlertLevel.CRITICAL]: 'CRIT',
+const ICONS: Record<number, ReactNode> = {
+  [AlertLevel.UNKNOWN]: <span style={{ opacity: 0.5 }}>?</span>,
+  [AlertLevel.INFO]: (
+    <svg width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)